### PR TITLE
Fix group of lights

### PR DESCRIPTION
**Pull request by fmeies**
A group of lights like this

```
Group:Switch:OR(ON,OFF) lightGroup ["Lighting"]
Color light1 (lightGroup) ["Lighting"]
Color light2 (lightGroup) ["Lighting"]
```
is not discovered correctly. In v2, the individual lights as well as the group of lights are discovered. In v3, only the individual lights are discovered.

I guess we have to distinguish between "endpoint" groups like e.g., Alexa.Endpoint.Thermostat or Alexa.Endpoint.Speaker and "simple" groups like a group of lights or rollershutter items.

What do you think?

### CODE REVIEW

**Comment by digitaldan** [comment]
Take a look at https://github.com/eclipse/smarthome/pull/4390#issuecomment-381591821 , I'm going to redo using tags for metadata instead.  This should not be too different, but I wonder if we can improve the group issue, i have not had time to look at it yet.  In any case we will still support v2 tags.  

Also I apologize for some sloppiness in the code with things commented out and random log statements, this was my personal branch and was not anticipating others to share, but I definitely welcome the help!

**Comment by fmeies** [comment]
Thank you for your feedback. I'll have a look at the referenced metadata discussion.

File: alexaContextProperties.js [hunk 1]
```diff
@@ -65,6 +65,12 @@ AlexaContextProperties.prototype.powerLevelStateProperty = function (state) {
  * @param {*} state 
  */
 AlexaContextProperties.prototype.colorStateProperty = function (state) {
+  // color item may receive brightness value
+  if (!isNaN(state)) {
+    var num = parseInt(state);
+    state = '0,0,' + num;
+  }
+
```
Comment (jsetton) [inline alexaContextProperties.js:73]: One of the changes I submitted was to use the latest openhab state opposed to the original requested state in Alexa response messages. If @digitaldan is fine with that, this change wouldn't be necessary.

c7e8a5a50b8f5ab936bee7f5d30b2ef8d35d2cbf

**Comment by digitaldan** [comment]
Thanks!  I'm going to merge this now and test, really appreciate the help.

**Pull request merged**